File: Semestres/11.Febrero-Junio 2022/MDII/corte2/MainArbol.java
public class MainArbol {

    public static void main(String[] args) {
        Arbol arbolObj = new Arbol(
                1,
                new Arbol(
                        2,
                        new Arbol(4, null, null),
                        new Arbol(5, null, null)),
                new Arbol(3,
                        new Arbol(6, null, null),
                        new Arbol(7, null, null)));

        System.out.println(arbolObj);

        Arbol arbolObj2 = new Arbol(
                1,
                new Arbol(
                        7,
                        new Arbol(2, null, null),
                        new Arbol(
                                6,
                                new Arbol(5, null, null),
                                new Arbol(11, null, null))),
                new Arbol(
                        9,
                        null,
                        new Arbol(
                                9,
                                new Arbol(5, null, null),
                                null)));

        System.out.println(arbolObj2.preorder());
        System.out.println(arbolObj2.inorder());
        System.out.println(arbolObj2.posorder());
    }

}

File: Semestres/11.Febrero-Junio 2022/MDII/corte2/MainArbol.py
from Arbol import Arbol


arbolObj = Arbol(
    1,
    Arbol(
        2,
        Arbol(4,None,None),
        Arbol(5,None,None),
    ),
    Arbol(3,
        Arbol(6,None,None),
        Arbol(7,None,None),
    )
)

arbolObj2 = Arbol(
    1,
    Arbol(
        7,
        Arbol(2,None,None),
        Arbol(
            6,
            Arbol(5,None,None),
            Arbol(11,None,None),
            )
        ),
    Arbol(
        9,
        None,
        Arbol(
            9,
            Arbol(5,None,None),
            None
        )
    )
)

print(arbolObj2.preorder())
print(arbolObj2.inorder())
print(arbolObj2.posorder())